src/applications/disability-benefits/all-claims/config/schema.js

```diff
@@ -717,6 +718,49 @@ const schema = {
         ],
       },
     },
+    ptsdIncident: {
+      type: 'object',
+      properties: {
+        date: { $ref: '#/definitions/date' },
+        description: { type: 'string' },
+      },
+    },
+    secondaryPtsdIncident: {
+      type: 'object',
+      properties: {
+        authorities: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              name: {
+                type: 'string',
+              },
+              address: {
+                type: 'object',
+                required: [],
+                properties: {
+                  ..._.omit(['addressLine3'], baseAddressDef.properties),
+                  country: {
+                    default: 'USA',
+                    type: 'string',
+                    enum: countries.map(object => object.value),
```

Comment:
What do you think about `s/object/country/g`?

Reply:
😬 yeah, 100%